adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -1013,7 +1013,7 @@ public ProtobufOutputStreamGenerator(final String outputPath) {
 
         public OutputStream getStream() throws IOException {
             OutputStream newStream = Files.newOutputStream(Paths.get(
-                    targetPath + File.separator + streamCounter +
+                    targetPath + "-" + streamCounter +
```

Comment:
Just out of curiosity: what is this a fix for and is there any reason we'd run into troubles using the `-` (like it's an invalid character on some file systems or something)

Reply:
Using `File.separator` caused an issue: the system could not find the path to `targetPath/streamcounter.pb/`.